Main.java

```diff
@@ -0,0 +1,54 @@
+class Car {
+private String brand;
+private String model;
+
+public Car(String brand, String model) {
+    this.brand = brand;
+    this.model = model;
+}
+
+public String getBrand() {
+    return brand;
+}
+
+public String getModel() {
+    return model;
+}
+}
+
+class CarShowroom {
+    private Car[] cars;

```

Comment:
Composition.
Read what is composition?
what is meant by composition over inheritance?

Reply:
which is preferred and why